packages/mesa.rb

```diff
@@ -26,9 +26,13 @@ class Mesa < Package
   depends_on 'libxtrans'
   depends_on 'libxkbfile'
   depends_on 'wayland'
+  depends_on 'python'
+  depends_on 'bison'
 
+  # tested on armv7l
   def self.build
-    system "./configure --enable-gles2 --with-egl-platforms=x11,wayland,drm --enable-gbm --enable-shared-glapi --with-gallium-drivers=swrast --disable-dri3"
+    system "pip install Mako"
```

Comment:
Please add `--prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}` options to the pip install line so the package can track this dependency and uninstall when the package is removed.